src/tools/pub/dependencies.md

```diff
@@ -460,9 +462,9 @@ this flag only resolves new dependencies if:
 * `pubspec.lock` is not missing
 * The packages' [content hashes][] match
 
-[`--enforce-lockfile`]: /tools/pub/cmd/pub-get#--enforce-lockfile
-[lockfile]: /tools/pub/glossary#lockfile
-[content hashes]: /tools/pub/glossary#content-hashes
+[`--enforce-lockfile`]: {{site.url}}/tools/pub/cmd/pub-get#--enforce-lockfile
```

Comment:
I haven't taken a look at this PR yet, but just scanning quickly, we don't use `{{site.url}}` on dart.dev. Just the base relative URLs.

Reply:
True, and I will remove. I think we should going forward to be consistent between both sites.

Reply:
That's the goal. I hope to make it no longer necessary on flutter/website soon.